src/docs/sdk/check-ins.mdx

```diff
@@ -24,6 +24,11 @@ The following fields exist:
 
 : **String, required**. Check-In ID (unique and client generated).
 
+  This may be provided as a empty UUID (128 bit zero value) to indicate to
+  Sentry that the checkin should update the previous check-in that is not in
+  the terminal state. If the most recent check-in is already in a terminal
+  state a new one will be created instead.
```

Comment:
It might be worth clarifying what terminal state means.

Reply:
yeah reverse the logic so it should "update the most recent check-in in an `in progress` state. If the most recent check-in is not `in progress`..."